tox.ini

```diff
@@ -74,6 +74,21 @@ image = redisfab/lots-of-pythons
 volumes =
     bind:rw:{toxinidir}:/data
 
+[docker:redis_cluster]
+name = redis_cluster
+image = barshaul/redis-py:6.2.6-cluster
+healtcheck_cmd = python -c "import socket;print(True) if 0 == socket.socket(socket.AF_INET, socket.SOCK_STREAM).connect_ex(('127.0.0.1',16379)) else False"
```

Comment:
If we're going to use all of these ports - perhaps a healthcheck that checks the entire cluster is up is more useful...